readme.txt

```diff
@@ -190,6 +190,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Tweak - Updated ARF upsell notice to only display in admin dashboard. [CT-67]
```

Comment:
We use the ARF acronym internally, we should use `Attendee Registration Fields` when it's something that'd be communicated to users.